tracee-ebpf/tracee/tracee.go

```diff
@@ -457,7 +463,34 @@ func (t *Tracee) initTailCall(tailNum uint32, mapName string, progName string) e
 
 	return err
 }
+func getAddrFromKallsyms(func_name string) (uint64, error) {
```

Comment:
same comments like for `getModuleOwnerBySymbol()` - about the errors and the loops.

Reply:
i think you should combine this function and the getModuleOwnerBySymbol() func - and pass an argument to tell which value you want to be returned, module_name or address.